omigami/omi_settings.py

```diff
@@ -0,0 +1,19 @@
+import confuse
+import os
+
+config = confuse.Configuration("omigami", __name__)
+
+CLASSYFIRE_URL = config["plotting"]["urls"]["classyfire"]
+NPCLASSIFIER_URL = config["plotting"]["urls"]["NPclassifier"]
+
+
+def get_credentials_path():
+    return os.path.expanduser("~") + "/.omigami/credentials"
```

Comment:
Just curious. Does this work on Windows?

Reply:
Yeah, that's the point of it x) - I haven't tested though